src/packages/cli/helpers/build.js

```diff
@@ -9,6 +9,24 @@ const { promisify } = require('util')
 const copyFile = promisify(fs.copyFile)
 const lineReplace = require('line-replace')
 
+let prismaClientPlugin = {
+  name: 'prisma-client-plugin',
+  setup(build) {
+    // Redirect all imports from `@prisma/client*` to `../prisma-client*`. This is needed because of Studio.
+    build.onResolve({ filter: /^@prisma\/client/ }, (args) => {
+      return {
+        path: require.resolve(
+          path.resolve(
+            __dirname,
+            '../',
+            args.path.replace('@prisma/client', 'prisma-client'),
+          ),
+        ),
+      }
+    })
+  },
+}
+
```

Comment:
Ideally, I'd like things like this to stay in Studio, but unfortunately, that can't work in this scenario.